rasa_core/processor.py

```diff
@@ -240,18 +240,9 @@ def _get_action(self, action_name):
 
     def _parse_message(self, message):
         # for testing - you can short-cut the NLU part with a message
-        # in the format _intent[entity1=val1,entity=val2]
+        # in the format /intent{"entity1": val1, "entity2": val2}
         # parse_data is a dict of intent & entities
-        if (message.text.startswith(INTENT_MESSAGE_PREFIX) or
-                message.text.startswith("_")):
-            if RegexInterpreter.is_using_deprecated_format(message.text):
-                warnings.warn(
-                        "Parsing messages with leading `_` is deprecated and "
-                        "will be removed. Instead, prepend your intents with "
-                        "`{0}`, e.g. `{0}mood_greet` "
-                        "or `{0}restart`.".format(INTENT_MESSAGE_PREFIX))
-            parse_data = RegexInterpreter().parse(message.text)
-        else:
+        if (message.text.startswith(INTENT_MESSAGE_PREFIX)):
```

Comment:
The intent-message case should still be parsed by the RegexInterpreter, no? Also, we still need the `else` case for a non-intent message

Reply:
Wow, that was a pretty glaring error on my part.  I'm not sure how the tests didn't pick up on that so I will write one